packages/cerebral-forms/package.json

```diff
@@ -4,10 +4,10 @@
   "description": "Signals, actions and state factories to create forms",
   "main": "lib/index.js",
   "scripts": {
-    "test": "../../node_modules/.bin/mocha --compilers js:../../node_modules/babel-register 'src/**/*.test.js'",
+    "test": "node ../../node_modules/mocha/bin/mocha --compilers js:../../node_modules/babel-register 'src/**/*.test.js'",
```

Comment:
Why are you changing ../../node_modules/.bin/mocha to ../../node_modules/mocha/bin/mocha?
It seems that you have in windows also the .bin directory, right?